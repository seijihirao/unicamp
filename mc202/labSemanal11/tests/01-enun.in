25
i 10 15
i 12 6
i 13 9
i 14 18
i 15 23
i 0 50
i 15 30
d 14 -7
d 10 8
m
m
m
m
m
m
m
t
