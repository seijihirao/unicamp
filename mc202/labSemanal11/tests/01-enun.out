minimo {14,-7}
minimo {12,6}
minimo {10,8}
minimo {13,9}
minimo {15,23}
minimo {0,50}
vazio
